src/test/java/org/esbtools/gateway/resync/ResyncErrorMessagesTest.java

```diff
@@ -0,0 +1,49 @@
+package org.esbtools.gateway.resync;
+
+import org.esbtools.gateway.resync.service.ResyncErrorMessages;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.Arrays;
+
+import static org.junit.Assert.assertEquals;
+
+public class ResyncErrorMessagesTest {
+    ResyncRequest resyncRequest;
+
+    @Before
+    public void setUp() {
+        resyncRequest = new ResyncRequest();
+        resyncRequest.setEntity("entity");
+        resyncRequest.setSystem("system");
+        resyncRequest.setKey("key");
+        resyncRequest.setValues(Arrays.asList("value1","value2"));
+    }
+
+    @After
+    public void tearDown() {
+        resyncRequest = null;
+    }
+
+    @Test
+    public void testInvalidSystem() throws Exception {
+        Assert.assertEquals(ResyncErrorMessages.INVALID_SYSTEM + "system", ResyncErrorMessages.invalidSystem("system"));
+    }
+
+    @Test
+    public void testIncompleteRequest() throws Exception {
+        assertEquals(ResyncErrorMessages.INCOMPLETE_REQUEST + resyncRequest, ResyncErrorMessages.incompleteRequest(resyncRequest));
+    }
+
+    @Test
+    public void testSystemNotConfigured() throws Exception {
+        assertEquals(ResyncErrorMessages.SYSTEM_NOT_CONFIGURED + "system", ResyncErrorMessages.systemNotConfigured("system"));
+    }
+
+    @Test
+    public void testResyncFailed() throws Exception {
+        assertEquals(ResyncErrorMessages.RESYNC_FAILED + resyncRequest, ResyncErrorMessages.resyncFailed(resyncRequest));
```

Comment:
If you really wanted to test these, I would personally avoid using the constants themselves. For instance, this didn't catch that the constant has the "%s" in it. On the other hand, I don't think it's terribly important to have that tested, though in a way nice for completeness sake.